src/main/scala/com/ing/wbaa/airlock/sts/api/ServerApi.scala

```diff
@@ -0,0 +1,29 @@
+package com.ing.wbaa.airlock.sts.api
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.Future
+import scala.util.{ Failure, Success }
+
+trait ServerApi extends LazyLogging {
+
+  protected[this] def checkConnection(): Future[Unit]
```

Comment:
Maybe change the name to `checkDbConnection` as now it's not really clear which connection is checked